main.go

```diff
@@ -87,6 +90,8 @@ func main() {
 
 			if err != nil {
 				zeroLogger.Error().Err(err).Msg("Failed to initialize Kafka consumers")
+				// If err is something then starconsumer again
```

Comment:
for a second I thought this was a TODO / future optimization, but now I'm realizing the kafka consumer / reads from `batchPipeline` aren't happening on the main thread.

as it stands would the process just keep running and never restart the consumers?

Reply:
Yes you're right. I didn't intend this as a TODO, but as a task. I'll complete it. Good catch.